### ISSUE TITLE
Compatibility with mongoid 6

### ISSUE THREAD

**Issue by phinfonet** [mongoid-rspec/mongoid-rspec#169]
Have any expectation to upgrade that to use with mongoid 6?

**Comment by rodrigopinto** [comment]
Hi, @phinfonet, like most of the maintainers, are not currently working with MongoDB, we are dependent on contributions. Feel free to send us a tested PR that we review and merge.

**Comment by phinfonet** [comment]
Hi, @rodrigopinto i'm sorry, but needed to do some monkey patches, but soon i'll try to make a PR.

**Comment by dbackeus** [comment]
+1

**Comment by michfarr** [comment]
it appears there are now several PR created for Mongoid 6 compatibility.
#179 
#175

**Comment by sahilchopra** [comment]
+1

**Comment by jschorr** [comment]
+1

**Comment by SaimonL** [comment]
At the time being I will go on without writing any test. Once it works  with MongoID 6 I will start writing tests again.

**Comment by SaimonL** [comment]
+1

**Comment by 90yukke** [comment]
@phinfonet beta support of mongoid 6 is already in master. See https://github.com/mongoid-rspec/mongoid-rspec/pull/179

In fact it just makes specs green. There're more things to do, but I'll create separate issues for that.

**Comment by gurix** [comment]
How bad is it to release a new version for mongoid 6 Support from master branch?
I struggle with this because on the 3.0 gem it said...
```
$ bundle update
Bundler could not find compatible versions for gem "mongoid":
  In Gemfile:
    mongoid (~> 6.0.0)

    mongoid-rspec (~> 3.0) was resolved to 3.0.0, which depends on
      mongoid (~> 5.0)
```
When I use the master branch `bundle-audit ` says....
```
$  bundle-audit
Insecure Source URI found: git://github.com/mongoid-rspec/mongoid-rspec.git
Vulnerabilities found!
```

**Comment by tagliala** [comment]
@gurix you can solve your issue by using 

```rb
gem 'mongoid-rspec', git: 'https://github.com/mongoid-rspec/mongoid-rspec.git'

```

instead of `gem 'mongoid-rspec', github: 'mongoid-rspec/mongoid-rspec'`

or by adding

```rb
git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
  "https://github.com/#{repo_name}.git"
end
```

at the beginning of your `Gemfile` (this is a [standard from rails 5.0 Gemfile template](https://github.com/rails/rails/blob/255ef67d5cd9acb823bce687c85b73e7f15accc8/railties/lib/rails/generators/rails/app/templates/Gemfile#L3))

@90yukke could you please consider reopening this issue until a mongoid 6.0 gem version will be released?
